modules/standard/BigInteger.chpl

```diff
@@ -5053,15 +5047,21 @@ module BigInteger {
   }
 
 
+  /* Computes the mod operator on the two arguments, defined as
+     ``mod(m,n) = m - n * floor(m / n)``.
```

Comment:
It'd be good to use the argument names instead of `m` and `n` in this description.